phpstan-baseline.neon

```
@@ -31,19 +31,19 @@ parameters:
 			path: src/DependencyInjection/SentryExtension.php
 
 		-
-			message: "#^Parameter \\$event of method Sentry\\\\SentryBundle\\\\EventListener\\\\ErrorListener\\:\\:onKernelException\\(\\) has invalid typehint type Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\\.$#"
+			message: "#^If condition is always true\\.$#"
```

Al those ignored error should be trimmed down... Maybe in a separate PR.

---

I have plans to refactorize the remaining tests so I expect a lof of these errors to disappear. However, there are some entries that are here because of PHPStan that doesn't know the code must run with different versions of the vendors and thus some things may be defined in a version and not in another and I don't think we can do anything about this